examples/next/package.json

```diff
@@ -15,6 +15,7 @@
     "@aws-amplify/ui-react-storage": "^1.2.3",
     "@aws-amplify/ui-react-notifications": "^1.0.0",
     "@aws-amplify/ui-react-geo": "^1.0.0",
+    "@aws-sdk/credential-providers": "3.321.1",
```

Comment:
Do you know why it has conflicting version with `@aws-sdk/types`?

Reply:
Hm this was not intentional, i can use matching versions

Reply:
Yep, that'll help reduce the bundle size!

Reply:
fixed!